scopedLogger.go

```diff
@@ -15,6 +16,15 @@ type ScopedLogger struct {
 	fields    Fields
 	logger    *Logger
 	formatter Formatter
+	mux       sync.Mutex
```

Comment:
mux is an unfortunate name for a mutex. Instead mux implies a MUltipleXer of some sort. Not go specific.

Reply:
Ok, I thought that `mux` is a "standand" people use, since I have seen in other packages, and even in the golang playground examples: https://tour.golang.org/concurrency/9 but I can definitely change.

Reply:
Changed to `mutex`